src/main/java/gift/entity/Product.java

```diff
@@ -25,6 +28,9 @@ public class Product {
     @Column(name = "image_url", nullable = false)
     private String imageUrl;
 
+    @OneToMany(mappedBy = "product", cascade = jakarta.persistence.CascadeType.ALL, orphanRemoval = true)
```

Comment:
import를 해주면 코드가 더 깔끔해질 것 같아요. 😃

```suggestion
    @OneToMany(mappedBy = "product", cascade = CascadeType.ALL, orphanRemoval = true)
```